fedora/gags.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from maubot import MessageEvent
+from maubot.handlers import command
+
+from .handler import Handler
+
+"""
+Old zodbot had a lot of fun little gags.
+Why not carry (some of) them over?
+"""
+
+
+class MiscHandler(Handler):
+    @command.new(name="fire")
+    @command.argument("firee", required=True)
+    async def fire(self, evt: MessageEvent, firee: str) -> None:
+        await evt.mark_read()
+        await evt.react("🔥")
+        await evt.reply(f"adamw fires {firee}")
```

Comment:
This should be `@adamwill:fedora.im`

Reply:
What does Adam think about being pinged each time someone uses this command?

Reply:
Well, it's @AdamWill's thing, we can ask. Though, he was pinged before when it was used.

Reply:
i think i'd rather not be pinged. thanks!